woodwork/tests/accessor/test_serialization.py

```diff
@@ -1,5 +1,6 @@
 import json
 import os
+import shutil
```

Comment:
Maybe this could be done in a separate PR, but shouldn't we be able to structure these tests in a way that we don't have to manually clean up the directory since `tmpdir` should be unique to the test invocation?